diff --git a/Explorer/Assets/DCL/Infrastructure/Global/Dynamic/WorldManifestProvider.cs b/Explorer/Assets/DCL/Infrastructure/Global/Dynamic/WorldManifestProvider.cs
index 7c38589d..f1275ded 100644
--- a/Explorer/Assets/DCL/Infrastructure/Global/Dynamic/WorldManifestProvider.cs
+++ b/Explorer/Assets/DCL/Infrastructure/Global/Dynamic/WorldManifestProvider.cs
@@ -23,7 +23,7 @@ namespace Global.Dynamic
 
         private static URLAddress ORG_MANIFEST_URL = URLAddress.FromString("https://places-dcf8abb.s3.amazonaws.com/WorldManifest.json");
         private static URLAddress ZONE_MANIFEST_URL = URLAddress.FromString("https://places-e22845c.s3.us-east-1.amazonaws.com/WorldManifest.json");
-        private static readonly string[] MAIN_REALM_NAMES = { "main", "shiva", "hela", "heimdallr", "baldr", "artemis", "loki", "dg", "hephaestus", "unicorn", "marvel", "nftworld" };
+        private static readonly string[] MAIN_REALM_NAMES = { "main", "shiva", "hela", "heimdallr", "baldr", "artemis", "loki", "dg", "hephaestus", "unicorn", "marvel", "nftworld", "myrealm" };
         private const string dclWorldName = "dcl.eth";
 
         private WorldManifest? cachedMainManifest;
